pkg/tms/tms_test.go

```diff
@@ -599,7 +599,7 @@ func TestSendRequest(t *testing.T) {
 func TestNewCommunicationInstance(t *testing.T) {
 	t.Run("test success", func(t *testing.T) {
 		uploaderMock := uploaderMock{responseBody: `{"token_type":"bearer","access_token":"testOAuthToken","expires_in":54321}`, httpStatusCode: http.StatusOK}
-		communicationInstance, err := NewCommunicationInstance(&uploaderMock, "https://tms.dummy.sap.com", "https://dummy.sap.com", "testClientId", "testClientSecret", false)
+		communicationInstance, err := NewCommunicationInstance(&uploaderMock, "https://tms.dummy.sap.com", "https://dummy.sap.com", "testClientId", "testClientSecret", false, piperHttp.ClientOptions{})
```

Comment:
Would it make sense to test for the custom description behaviour? Checking that it's "Created by Piper" on empty, and that it is what the user sets when not empty.

Reply:
Not possible with current implementation, but will be possible to test it with the integration test, see https://github.com/SAP/jenkins-library/pull/4134/commits/156d8ca6112bfb801b90ed995dfe48420b73b003